Makefile

```diff
@@ -21,10 +21,12 @@ GO_SYSTEM_VERSION ?= "$(shell go version | awk '{ gsub(/go/, "", $$3); print $$3
 BEE_API_VERSION ?= "$(shell grep '^  version:' openapi/Swarm.yaml | awk '{print $$2}')"
 BEE_DEBUG_API_VERSION ?= "$(shell grep '^  version:' openapi/SwarmDebug.yaml | awk '{print $$2}')"
 
+VERSION ?= "$(shell git describe --tags --abbrev=0 | cut -c2-)"
```

Comment:
The most accurate way of getting the latest tag should be the following: `git describe --tags $(git rev-list --tags --max-count=1)`